cves/CVE-2003-0132.yml

```diff
@@ -3,15 +3,15 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: CWE-789: Uncontrolled Memory Allocation
```

Comment:
Please just say CWE-789 or 789. This will fix the CI

Reply:
Thanks, will do